bundles/org.openhab.binding.hyperion/src/main/java/org/openhab/binding/hyperion/internal/handler/HyperionNgHandler.java

```diff
@@ -295,6 +297,9 @@ private void updateComponents(List<Component> components) {
                 case COMPONENT_LEDDEVICE:
                     updateState(CHANNEL_LEDDEVICE, componentState);
                     break;
+                case COMPONENT_ALL:
```

Comment:
How does the ALL component only correspond to the hyperion enabled channel? Is there no separate enabled channel?

Reply:
According to [Hyperion docs](https://docs.hyperion-project.org/en/json/ServerInfo.html), sending on/off to special component "ALL":
- saves/restores current state of every component,
- toggles every component on/off,
effectively disabling/enabling hyperion.

Also I double checked by looking at their standalone app which comes with Hyperion:
```
$ hyperion-remote --help | grep -e Activate -e Deactivate
  --off                                              Deactivates hyperion
  --on                                               Activates hyperion
```

What this app do is actually calling API for component "ALL":
```
$ strace -s 9999 -f -e trace=network,write hyperion-remote --off -a 192.168.0.203:19444 
...
write(2, "Connected to: \"192.168.0.203:19444\"\n", 36Connected to: "192.168.0.203:19444"
...
write(2, "Disable Component \"ALL\"\n", 24Disable Component "ALL"
...
write(5, "{\"command\":\"componentstate\",\"componentstate\":{\"component\":\"ALL\",\"state\":false}}\n", 80) = 80
...
```

Basically, there is no dedicated disable/enable channel except API shorthand for disabling/enabling ALL components.

State updates for HyperionEnabled channel look like this in OH logs:
![Screenshot_20200615_195229](https://user-images.githubusercontent.com/66778456/84689820-d0db7600-af41-11ea-84d3-2cfce7e9f4be.png)